Hashids/Hashids.swift

```diff
@@ -305,7 +307,7 @@ func transform<T: CollectionType where T.Generator.Element: Equatable>(a: T, _ b
 }
 
 func unique<T: CollectionType where T.Generator.Element: Equatable>(a: T) -> [T.Generator.Element] {
-    return transform(a, a) { (var c, a, b, e) in
+    return transform(a, a) { (inout c: [T.Generator.Element], a, b, e) in
```

Comment:
nope, I didn't have to do anything but delete the var apparently, the transform method passed back the type properly I just didn't trust it originally I guess